modules/apps/frontend-taglib/frontend-taglib-sample-web/src/main/resources/META-INF/resources/partials/search_iterator.jsp

```diff
@@ -0,0 +1,39 @@
+<%--
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+--%>
+
+<%@ include file="/init.jsp" %>
+
+<%
+SearchIteratorDisplayContext searchIteratorDisplayContext = (SearchIteratorDisplayContext)request.getAttribute(SamplePortletKeys.SEARCH_ITERATOR_DISPLAY_CONTEXT);
+%>
+
+<clay:container-fluid>
+	<liferay-ui:search-container
+		searchContainer="<%= searchIteratorDisplayContext.getSearchContainer() %>"
+	>
+		<liferay-ui:search-container-row
```

Comment:
The tag we are testing is `<liferay-ui:search-container>`. Shoudn't it be `search_container.jsp`, `SearchContainerDisplayContext` etc?

Reply:
Actually, the tag we are testing is search-iterator which needs to be wrapped by a search-container. If you look at the bottom of that file you'll see we are adding liferay-ui:search-iterator tag.